Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Eel/ElasticSearchQueryBuilder.php

```diff
@@ -449,6 +468,28 @@ public function fulltext($searchWord) {
 	}
 
 	/**
+	 * Wrap search term matches found in the result for highlighting purposes. Only available for fulltext searches.
```

Comment:
don't think so since the output changes because of it and that might not be desired